jupyter_conda/handlers.py

```diff
@@ -349,12 +349,17 @@ async def get(self):
         """`GET /packages` Search for packages.
         
         Query arguments:
+            package (str): optional package name to seach dependencies
             query (str): optional string query
         """
+        pkg = self.get_query_argument("package", "")
```

Comment:
I would rather use a query arguments like `dependencies` equals to 0 or 1 to decide if we need to search or to  get the dependencies (with default value to false).
See for example `PackagesEnvironmentHandler.post`